README.md

```diff
@@ -130,4 +130,17 @@ This rule checks for cases where `@import` is called multiple times with the sam
 ### Command line
 ```bash
 ziglint --dupe-import
+```
+
+## `file_as_struct`
+This rule checks for file name capitalization in the presence of top level fields.
```

Comment:
It might be a good idea to elaborate a little more on the rule here (ie: if there are top level fields the file name should be capitalized).